index.html

```diff
@@ -1,48 +1,102 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  
-    <link rel="stylesheet" href="dashboard.css">
+  <head>
     <title>EMS - Dashboard</title>
     <script src="https://cdn.tailwindcss.com"></script>
-</head>
-<body>
-    <div class="nav h-16">
-        <div class="logo">Employee Management System</div>
-        <ul class="nav-links">
-            <li><a href="#">Home</a></li>
-            <li><a href="#">Employees</a></li>
-            <li><a href="#">Reports</a></li>
-            <li><a href="#">Settings</a></li>
-        </ul>
-        <div class="user-menu  ">
-            <span>Admin</span> | <a href="/login.html">Logout</a>
-        </div>
+    <style type="text/tailwindcss">
+      @layer utilities {
+  
+  body {
+    background-color: rgb(255, 255, 255);
+  }
+  .nav-links li {
+  margin: 0 15px;
+}
+
+.nav-links a {
+  color: #fff;
+  text-decoration: none;
+  font-size: 16px;
+}
+
+
+
+.user-menu a {
+  color: #fff;
+  text-decoration: none;
+  margin-left: 10px;
+}
+
+
+
+.dashboard-header h1 {
+  font-size: 28px;
+  margin-bottom: 20px;
+}
+
+
+
+
+
+.card h2 {
+  font-size: 22px;
+  margin-bottom: 10px;
+}
+
+.card p {
+  font-size: 18px;
+  font-weight: bold;
+}
+
+.card ul {
+  list-style-type: none;
+  padding: 0;
+}
+
+.card ul li {
+  margin-bottom: 10px;
+}
+
+}
+    </style>
```

Comment:
@dev-gauravbahuguna, can we use separate CSS file to store CSS related code. 
Like one Common CSS file for common CSS usage, and other CSS files should be separatee as per each module.

Reply:
@amitpjoshiem  actually its not a css file , its part of tailwindcss and since we are using cdn at the moment , i had to include it in the head tag

Reply:
seperate css file made